db/R/dbfiles.R

```diff
@@ -34,16 +34,17 @@
 dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate, mimetype, formatname, parentid=NA, con, hostname=fqdn()) {
   
   name <- basename(in.path)
-  filename <- file.path(in.path,in.prefix)
+  filename <- file.path(in.path, in.prefix)
   
   if (hostname == "localhost") hostname <- fqdn();
   
+  mimetypeid <- db.query(paste0('SELECT id FROM mimetypes where type_string = ', mimetype, ';'))
```

Comment:
I have gone ahead and inserted a new record here, but in posteriors I throw an error.
 
There are 418 records in mimetypes - should we allow users to insert arbitrary strings, or should we fail if it is not recognized? Otherwise we could end up with lots of duplicates and a mess like the variables table.